app/components/autocomplete/at_mention/index.js

```diff
@@ -9,13 +9,19 @@ import {autocompleteUsers} from 'mattermost-redux/actions/users';
 import {getDefaultChannel} from 'mattermost-redux/selectors/entities/channels';
 import {getProfilesInCurrentChannel, getProfilesNotInCurrentChannel} from 'mattermost-redux/selectors/entities/users';
 
+import {ViewTypes} from 'app/constants';
 import AtMention from './at_mention';
 
 function mapStateToProps(state, ownProps) {
     const {currentChannelId} = state.entities.channels;
 
     let postDraft;
-    if (ownProps.rootId.length) {
+    if (ownProps.isSearch) {
+        const searchDraft = state.views.channel.drafts[ViewTypes.SEARCH];
```

Comment:
Maybe we should pass the draft in as a prop instead of having to figure out where in the store to get it from? The parent of this component should have the draft available since it's rendering the textbox